### PR TITLE
Updated doc pages for Docker, Kafka, Nginx ST's

### PR DESCRIPTION
**Pull request by yasar-sumologic**
## Purpose of this pull request

This pull request adds Docker, Kafka, Nginx STs

## Select the type of change

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions, updating sections
- [X] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - .clabot, version updates, maintenance, dependencies, new packages for the site (Docusaurus, Gatsby, React, etc.)

### CODE REVIEW

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+
+### Step 2: Configure the source template
+
+In this step, you will configure the yaml required for Docker Collection. Below are the inputs required for configuration :
+
+- **Name**. Name of the source template.
+- **Description**. Description for the source template.	
+- **Docker Event log location**. Enter the path of the JSON file generated through the command in the prerequisite section.
+- **Endpoint**. Address to reach the desired Docker daemon (default: `unix:///var/run/docker.sock`).
+- **Excluded Image List**. A list of strings, [regexes](https://golang.org/pkg/regexp/), or [globs](https://github.com/gobwas/glob) whose referent container image names will not be among the queried containers for scrapping metrics. Learn more about [*excluded_images*](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md#configuration).
```
Comment (sumoanema) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:72]: Add Fields/Metadata user input

Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:72]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+		
+### Step 1: Set up remotely managed OpenTelemetry collector
+		
+import CollectorInstallation from '../../../../reuse/apps/opentelemetry/collector-installation.md';
+		
+<CollectorInstallation/>
+
+### Step 2: Configure the source template
+
+In this step, you will configure the yaml required for Docker Collection. Below are the inputs required for configuration :
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:66]:
```suggestion
In this step, you will configure the yaml required for Docker Collection. Below are the inputs required for configuration:
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+		
+### Step 1: Set up remotely managed OpenTelemetry collector
+		
+import CollectorInstallation from '../../../../reuse/apps/opentelemetry/collector-installation.md';
+		
+<CollectorInstallation/>
+		
+### Step 2: Configure the source template
+		
+In this step, you will configure the yaml required for Nginx Collection. Below are the inputs required for configuration :
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md:76]:
```suggestion
In this step, you will configure the yaml required for Nginx collection. Below are the inputs required for configuration:
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+
+- **Name**. Name of the source template.
+- **Description**. Description for the source template.	
+- **Endpoint**. The URL of the httpd status endpoint (default: `http://localhost:80/status`).
+- **Path to Nginx access Log file**. Enter the path to the Access log file for your Nginx instance.
+- **Path to Nginx Error Log file**. Enter the path to the error log file for your Nginx instance.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md:82]:
```suggestion
- **Path to Nginx error Log file**. Enter the path to the error log file for your Nginx instance.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+If not already present, the following [Fields](/docs/manage/fields/) are created as part of Source template creation.
+
+- **`sumo.datasource`**. Fixed value of **docker**. 
+- **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the docker env resides, such as `dev`, `prod`, or `qa`.
+
+## Prerequisites
+
+This section provides instructions for configuring metrics and log collection for the Sumo Logic app for Docker.
+
+#### For Metric collection
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:33]:
```suggestion
#### For metrics collection
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+
+Metrics are collected through the [Docker Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md) of OpenTelemetry. This requires Docker API version 1.22+ and only Linux is supported.
+
+#### For Log collection
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:37]:
```suggestion
#### For logs collection
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+
+To collect the Docker container event logs, the following command needs to be executed on the host machine and needs to be kept running, for monitoring all the Docker container-related events. The following command also needs a JSON file path where these container events can be dumped.
+
+```
+docker events -f 'type=container' --format '{{json .}}' > <PATH_TO_JSON> & disown
+```
+Path to this JSON file will be required in the [next step](#step-2-configure-integration), where events are sent to Sumo Logic through a filelog receiver and seen as part of the **Docker - Overview** dashboard. Also, you can add additional parameters to this command to send events for specific containers. [Learn more](https://docs.docker.com/engine/reference/commandline/events/).
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:44]:
```suggestion

The path to this JSON file will be required in the [next step](#step-2-configure-integration), where events are sent to Sumo Logic through a filelog receiver and seen as part of the **Docker - Overview** dashboard. Also, you can add additional parameters to this command to send events for specific containers. [Learn more](https://docs.docker.com/engine/reference/commandline/events/).
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md (hunk 2 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md (hunk 3 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md (hunk 4 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md (hunk 5 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md (hunk 6 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md (hunk 7 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+### Step 2: Configure the source template
+
+In this step, you will configure the yaml required for Docker Collection. Below are the inputs required for configuration:
+
+- **Name**. Name of the source template.
+- **Description**. Description for the source template.	
+- **Docker Event log location**. Enter the path of the JSON file generated through the command in the prerequisite section.
+- **Endpoint**. Address to reach the desired Docker daemon (default: `unix:///var/run/docker.sock`).
+- **Excluded Image List**. A list of strings, [regexes](https://golang.org/pkg/regexp/), or [globs](https://github.com/gobwas/glob) whose referent container image names will not be among the queried containers for scrapping metrics. Learn more about [*excluded_images*](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md#configuration).
+- **Fields/Metadata**. You can provide any customer fields to be tagged with the data collected. By default sumo tags `_sourceCategory` with the value otel/kafka.
```
Comment (sumoanema) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:73]: otel/kafka ? should be otel/docker

Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:73]: fixed it.

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/docker-icon.png')} alt="Thumbnail icon" width="100"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="30"/>
+
+Docker source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of logs and metrics of Docker to Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:20]:
```suggestion
The Docker source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent, you can ensure collection of logs and metrics of Docker to Sumo Logic.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+## Fields creation in Sumo Logic for Docker
+
+If not already present, the following [Fields](/docs/manage/fields/) are created as part of Source template creation.
+
+- **`sumo.datasource`**. Fixed value of **docker**. 
+- **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the docker env resides, such as `dev`, `prod`, or `qa`.
+
+## Prerequisites
+
+This section provides instructions for configuring metrics and log collection for the Sumo Logic app for Docker.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:31]:
```suggestion
This section provides instructions for configuring metrics and log collection for the Sumo Logic Docker app.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+
+#### For metrics collection
+
+Metrics are collected through the [Docker Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md) of OpenTelemetry. This requires Docker API version 1.22+ and only Linux is supported.
+
+#### For logs collection
+
+To collect the Docker container event logs, the following command needs to be executed on the host machine and needs to be kept running, for monitoring all the Docker container-related events. The following command also needs a JSON file path where these container events can be dumped.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:39]:
```suggestion
To collect Docker container event logs, execute the following command on the host machine and keep it running to monitor all Docker container-related events. The command requires a JSON file path where these container events will be stored.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+### Step 2: Configure the source template
+
+In this step, you will configure the yaml required for Docker Collection. Below are the inputs required for configuration:
+
+- **Name**. Name of the source template.
+- **Description**. Description for the source template.	
+- **Docker Event log location**. Enter the path of the JSON file generated through the command in the prerequisite section.
+- **Endpoint**. Address to reach the desired Docker daemon (default: `unix:///var/run/docker.sock`).
+- **Excluded Image List**. A list of strings, [regexes](https://golang.org/pkg/regexp/), or [globs](https://github.com/gobwas/glob) whose referent container image names will not be among the queried containers for scrapping metrics. Learn more about [*excluded_images*](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md#configuration).
+- **Fields/Metadata**. You can provide any customer fields to be tagged with the data collected. By default sumo tags `_sourceCategory` with the value otel/docker.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:73]:
```suggestion
- **Fields/Metadata**. You can provide any customer fields to be tagged with the data collected. By default, Sumo Logic tags `_sourceCategory` with the value otel/docker.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md [hunk 13]
```diff
@@ -0,0 +1,85 @@
...
+Metrics are collected through the [Docker Stats Receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/main/receiver/dockerstatsreceiver/README.md) of OpenTelemetry. This requires Docker API version 1.22+ and only Linux is supported.
+
+#### For logs collection
+
+To collect the Docker container event logs, the following command needs to be executed on the host machine and needs to be kept running, for monitoring all the Docker container-related events. The following command also needs a JSON file path where these container events can be dumped.
+
+```
+docker events -f 'type=container' --format '{{json .}}' > <PATH_TO_JSON> & disown
+```
+The path to this JSON file will be required in the [next step](#step-2-configure-integration), where events are sent to Sumo Logic through a filelog receiver and seen as part of the **Docker - Overview** dashboard. Also, you can add additional parameters to this command to send events for specific containers. [Learn more](https://docs.docker.com/engine/reference/commandline/events/).
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md:44]: This link doesn't work: `[next step](#step-2-configure-integration)`

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 14]
```diff
@@ -0,0 +1,87 @@
...
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/integrations/containers-orchestration/kafka.png')} alt="icon" width="90"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="45"/>
+
+Kafka source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of logs and metrics of Kafka to Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:20]:
```suggestion
The Kafka source template generates an OpenTelemetry configuration that can be sent to a remotely managed OpenTelemetry collector (otelcol). By creating this source template and pushing the configuration to the appropriate OpenTelemetry agent, you can ensure the collection of Kafka logs and metrics in Sumo Logic.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 15]
```diff
@@ -0,0 +1,87 @@
...
+
+## Fields Creation in Sumo Logic for Kafka
+
+If not already present, the following [Fields](/docs/manage/fields/) are created as part of Source template creation.
+
+- **`sumo.datasource`**. Fixed value of **kafka**.
+- **`messaging.system`**. Fixed value of **kafka**.
+- **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the kafka env resides, such as `dev`, `prod`, or `qa`.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:28]:
```suggestion
- **`deployment.environment`**. This is a user-configured field set at the time of collector installation. It identifies the environment where the Kafka env resides, such as `dev`, `prod`, or `qa`.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 16]
```diff
@@ -0,0 +1,87 @@
...
+- **`messaging.cluster.name`**. User configured. Enter a name to uniquely identify your Kafka cluster. This cluster name will be shown in the Sumo Logic dashboards.
+- **`messaging.node.name`**. Includes the value of the hostname of the machine which is being monitored.
+
+## Prerequisites
+
+### For metrics collection
+
+Kafka metrics [receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kafkametricsreceiver) collects Kafka metrics (brokers, topics, partitions, and consumer groups) from the Kafka server. This app has been tested with following Kafka versions: 2.X and 3.X
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:36]:
```suggestion
The Kafka metrics [receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/kafkametricsreceiver) collects Kafka metrics (brokers, topics, partitions, and consumer groups) from the Kafka server. This app has been tested with following Kafka versions: 2.x and 3.x.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 17]
```diff
@@ -0,0 +1,87 @@
...
+
+### For logs collection
+
+Configure logging in Kafka: By default, Kafka logs (`server.log` and `controller.log`) are stored in the directory called `/opt/Kafka/kafka_<VERSION>/logs`. Make a note of this logs directory.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:40]:
```suggestion
In this section, you'll configure logging in Kafka. By default, Kafka logs (`server.log` and `controller.log`) are stored in the directory called `/opt/Kafka/kafka_<VERSION>/logs`. Make a note of this logs directory.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 18]
```diff
@@ -0,0 +1,87 @@
...
+
+### Step 1: Step 1: Set up remotely managed OpenTelemetry collector
+
+import CollectorInstallation from '../../../../reuse/apps/opentelemetry/collector-installation.md';
+
+<CollectorInstallation/>
+
+### Step 2: Configure the source template
+
+In this step, you will configure the yaml required for Kafka Collection. Below are the inputs required for configuration :
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:68]:
```suggestion
In this step, you will configure the yaml required for Kafka collection. Below are the inputs required for configuration:
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 19]
```diff
@@ -0,0 +1,87 @@
...
+
+- **Name**. Name of the source template.
+- **Description**. Description for the source template.	
+- **Server File log Path**. Enter the path to the Server log file for your Kafka instance.
+- **Controller file log path**. Enter the path to the Controller log file for your Kafka instance.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:73]:
```suggestion
- **Server File log Path**. Enter the path to the server log file for your Kafka instance.
- **Controller file log path**. Enter the path to the controller log file for your Kafka instance.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 20]
```diff
@@ -0,0 +1,87 @@
...
+- **Endpoint**. The URL of the broker endpoint (default: `localhost:9092`).
+- **Fields/Metadata**. You can provide any customer fields to be tagged with the data collected. By default sumo tags `_sourceCategory` with the value otel/kafka user needs to provide the value for `webengine.cluster.name`.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:75]:
```suggestion
- **Fields/Metadata**. You can provide any customer fields to be tagged with the data collected. By default, Sumo Logic tags `_sourceCategory` with the value otel/Kafka user needs to provide the value for `webengine.cluster.name`.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md [hunk 21]
```diff
@@ -0,0 +1,95 @@
...
+
+<head>
+  <meta name="robots" content="noindex" />
+</head>
+
+<p><a href="/docs/beta"><span className="beta">Beta</span></a></p>
+
+<img src={useBaseUrl('img/integrations/web-servers/nginx.png')} alt="Thumbnail icon" width="100"/> <img src={useBaseUrl('img/send-data/otel-color.svg')} alt="Thumbnail icon" width="30"/>
+
+Nginx source template creates an OpenTelemetry configuration that can be pushed to a remotely managed OpenTelemetry collector (abbreviated as otelcol). By creating this source template and pushing the config to the appropriate OpenTelemetry agent you can ensure collection of logs and metrics of Nginx to Sumo Logic.
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md:20]:
```suggestion
The Nginx source template generates an OpenTelemetry configuration that can be sent to a remotely managed OpenTelemetry collector (otelcol). By creating this source template and pushing the configuration to the appropriate OpenTelemetry agent, you can ensure the collection of Nginx logs and metrics in Sumo Logic.
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+		
+import OtelWindowsLogPrereq from '../../../../reuse/apps/opentelemetry/log-collection-prerequisite-windows.md';
+
+<OtelWindowsLogPrereq/>
+
+## Source template configuration
+
+You can follow the below steps to set a remotely managed OpenTelemetry collector and push the source template to it.
+
+### Step 1: Step 1: Set up remotely managed OpenTelemetry collector
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/kafka.md:60]:
```suggestion
### Step 1: Set up remotely managed OpenTelemetry collector
```

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md [hunk 23]
```diff
@@ -0,0 +1,95 @@
...
+### For metrics collection
+		
+The receiver gets stats from an nginx Web Server instance using the `/status` endpoint. This receiver supports Nginx Web Server version 1.26.1+.
+
+Receive server statistics, by configuring the server's `nginx.conf` file to [enable status support](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/nginxreceiver#configuration).
+		
+### For logs collection
+		
+Configure the Nginx access and error log files:
+1. Configure the logging of access logs and error logs via the instructions described in their [documentation](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/).
```
Comment (kimsauce) [inline docs/send-data/opentelemetry-collector/remote-management/source-templates/nginx.md:43]:
```suggestion
1. Configure the logging of access logs and error logs by following the instructions in their [documentation](https://docs.nginx.com/nginx/admin-guide/monitoring/logging/).
```

**Review by kimsauce — CHANGES_REQUESTED** [review]
Had some more changes!

File: docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md (hunk 13 above)
Comment (yasar-sumologic) [reply docs/send-data/opentelemetry-collector/remote-management/source-templates/docker.md]: removed that link.

**Comment by yasar-sumologic** [comment]
> Had some more changes!

Hi @kimsauce 
I've addressed your feedback on PR and believe it's ready for re-review; could you please take another look?

**Review by kimsauce — APPROVED** [review]

**Pull request merged**